src/lib/utils/escapeRefinement.ts

```diff
@@ -1,3 +1,5 @@
+function escapeRefinement(value: number): string | number;
+function escapeRefinement(value: string): string;
```

Comment:
1. I think I would have changed the function so it always returns a string just for simplicity.
2. isn't the return type on the function declaration `function escapeRefinement(value: string | number): string | number` is superfluous ?

Reply:
that's a good point, if this function is ever touched again, it could return a string in all cases

the last declaration isn't superfluous, you need a "overarching" function if you're doing overrides